src/ai/backend/manager/idle.py

```diff
@@ -302,22 +318,58 @@ async def check_session(self, session: Row, dbconn: SAConnection) -> bool:
 
 class UtilizationIdleChecker(BaseIdleChecker):
     """
-    Checks the idleness of a session by the current utilization of all
-    compute devices and agents assigned to it.
+    Checks the idleness of a session by the average utilization of compute devices.
     """
 
     name: ClassVar[str] = "utilization"
-    default_cpu_util_threshold: ClassVar[float] = 30.0
-    default_accelerator_util_threshold: ClassVar[float] = 10.0
-    default_initial_grace_period: ClassVar[float] = 300.0  # allow first 5 minutes to be idle
+    check_period: float = 20.0
```

Comment:
`check_interval` would be better than `check_period`.
Period usually means the time length of an action, while interval usually means the time length between two actions.
(e.g., I visit here for 1 week every 1 month. -> period = 1 week, interval = 1 month)